packages/server/dataloader/customLoaderMakers.ts

```diff
@@ -955,3 +964,59 @@ export const publicTemplatesByType = (parent: RootDataLoader) => {
     }
   )
 }
+
+export const allFeatureFlags = (parent: RootDataLoader) => {
+  return new DataLoader<'Organization' | 'Team' | 'User' | 'all', FeatureFlag[], string>(
+    async (scopes) => {
+      const pg = getKysely()
+      return await Promise.all(
+        scopes.map(async (scope) => {
+          let query = pg.selectFrom('FeatureFlag').selectAll().where('expiresAt', '>', new Date())
+          if (scope !== 'all') {
+            query = query.where('scope', '=', scope)
```

Comment:
+1 you can use the `.$if` method to make this a const if you want, but end of the day it's all the same